### PR TITLE
Virtual File System (Pipeline results)

### PR DESCRIPTION
**Pull request by awtkns**
@KlamChowder1 this is the base branch for the resulting file system.

### CODE REVIEW

**Comment by KlamChowder1** [comment]
@awtkns what do you think about putting it here? and should I add the delete files stuff like in the pipeline results table? 

image

**Comment by KlamChowder1** [comment]
- added the delete file stuff from pipeline runs, bc might have tons of files
- changed the "delete confirm" icon instead of "save", what do you think? 
vfs_download
vfs_delete

File: backend/api/models/user.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ class UserLDAP(Base):
     title = Column(String)
     department = Column(String)
     company = Column(String)
+    # has_approval = Column(Boolean, default=False)
```
Comment (asim-shrestha) [inline backend/api/models/user.py:49]: Whats the plan with this?

File: backend/api/pipelining/_tasks/run.py [hunk 2]
```diff
@@ -107,3 +107,19 @@ def dicom_output_task(run_id: int, node_id: int, previous_job_id: int = None):
...
+
+
+def post_run_cleanup(db, job: PipelineJob):
+    print('Pipeline finished')
+    mark_run_complete(db, job)
+
+    run: PipelineRun = job.run
+    output_path = pathlib.Path(run.get_abs_output_path())
+    for file in output_path.iterdir():
+        print(str(file))
```
Comment (asim-shrestha) [inline backend/api/pipelining/_tasks/run.py:119]:
```suggestion
```

**Review by asim-shrestha — APPROVED** [review]

File: backend/api/models/user.py (hunk 1 above)
Comment (KlamChowder1) [reply backend/api/models/user.py:49]: not sure.. may have to ask @awtkns about it

**Pull request merged**